scripts/release.js

```diff
@@ -136,3 +170,16 @@ function goBackToDevelop() {
 function build() {
   shell.exec('yarn run build');
 }
+
+function updateCanaryVersion(cb) {
+  const lastCommitHash = shell.exec('git rev-parse --short HEAD', {silent: true}).toString().trim();
+  const canaryVersion = `0.0.0-${lastCommitHash}`;
```

Comment:
as mentioned, this isn't incrementing, although I'm not too sure if that's an issue

Reply:
I've tested on a package and it seems to work as expected. I manage to publish three versions on the `canary` tag with three different commit messages. I've checked that the hash does not follow each other in any way. It looks like it's working.

> https://yarnpkg.com/en/package/@samouss/useless